indigo_libs/indigo/indigo_client.h

```diff
@@ -127,6 +128,7 @@ void indigo_service_name(const char *host, int port, char *name);
 /** Connect and start thread for remote server.
  */
 extern indigo_result indigo_connect_server(const char *name, const char *host, int port, indigo_server_entry **server);
+extern indigo_result indigo_connect_server_with_interface(const char *name, const char *host, int port, uint32_t interfaceIndex, indigo_server_entry **server);
```

Comment:
We do not use camel case, please rename to interface_index

Reply:
changed to interface_index